mapping/process-level-procurement.csv

```diff
@@ -3,7 +3,7 @@ Procuring entity,Enter name of the organisation carrying out the procurement,Pro
 Procuring entity contact details,Postal and Electronic address,Project Level: Publish the postal address in ``parties.address`` and the electronic address in ``parties.contactPoint``,OCDS: Check for a party with 'procuringEntity' in ``.roles`` and copy from ``.address`` and ``.contactPoint``,
 Contract administrative entity,Enter name of the organisation carrying out the contract administrative entity if different from the Procuring Entity,Project Level: Add an entry to ``parties`` with 'administativeEntity' included in its ``.roles`` | Contracting process: Record the name and identifier in ``.summary.tender.administrativeEntity``,OCDS: Check for a party with 'administrativeEntity' in ``.roles`` and copy from ``.name`` and ``.identifier``,
 Contract status,"Select from pre-award, active or closed",Contracting process: Publish in ``.summary.status`` using the contractingProcessStatus codelist,OCDS: Map following business logic in the contractProcessStatus codelist,`Issue #1 <https://github.com/open-contracting/infrastructure/issues/1>`_
```

Comment:
> Map following the business logic in the contractingProcessStatus codelist

Fix codelist name and add "the".

Reply:
updated in https://github.com/open-contracting/infrastructure/pull/103/commits/bc09d92dfd57f35d39e7feb708a582c3e135f07e